golang/cosmos/x/swingset/config.go

```diff
@@ -0,0 +1,45 @@
+package swingset
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"path/filepath"
+)
+
+const (
+	ConfigPrefix = "swingset"
+	FlagSlogfile = "swingset.slogfile"
+)
+
+// DefaultConfigTemplate defines a default TOML configuration section for the SwingSet VM.
+// See https://github.com/cosmos/cosmos-sdk/issues/20097 for auto-synchronization ideas.
+const DefaultConfigTemplate = `
+
+[swingset]
+# slogfile is the absolute path at which a SwingSet log "slog" file should be written.
+slogfile = ""
```

Comment:
Why not use the template to apply the default value here? I don't expect it'll ever change but that'd be a good example for other configs

```suggestion
slogfile = "{{ .SwingsetConfig.SlogFile }}"
```

Reply:
Done.